src/converter.ts

```diff
@@ -343,9 +347,13 @@ export class Converter {
       return baseFile.saveTmpFile('.html')
     })()
 
-    const uri = tmpFile
-      ? `file://${tmpFile.path}`
-      : `data:text/html;base64,${baseFile.buffer!.toString('base64')}`
+    const uri = await (async () => {
+      if (tmpFile) {
+        if (isWSL()) return `file:${await resolveWSLPath(tmpFile.path)}`
```

Comment:
Puppeteer is using Windows Chrome, so we have to resolve the file path for WSL if using tmpfile while conversion by enabling `--allow-local-files`.